mne/io/eeglab/eeglab.py

```diff
@@ -215,16 +235,30 @@ class RawEEGLAB(_BaseRaw):
         Path or instance of montage containing electrode positions.
         If None, sensor locations are (0,0,0). See the documentation of
         :func:`mne.channels.read_montage` for more information.
+    eog : list | tuple | 'auto'
+        Names or indices of channels that should be designated
+        EOG channels. If 'auto', the channel names containing
+        ``EOG`` or ``EYE`` are used. Defaults to empty tuple.
+    event_id : dict | None
+        The ids of the events to consider. If None (default),
+        an empty dict is used and `event_id_func` (see below) is called on
+        every event value. If dict, the keys will be mapped to trigger values
+        on the stimulus channel and only keys not in `event_id` will be handled
+        by `event_id_func`.
+        Keys are case-sensitive.
+        Example: {'SyncStatus': 1; 'Pulse Artifact': 3}.
+    event_id_func : None | callable
+        What to do for events not found in `event_id`. Must
+        take one `str` argument and return an `int`. If `None`, defaults to
+        `strip_non_integer`, which strips event codes such as "D128" or "S  1"
+        of their non-integer parts and returns the integer. Any event that is
+        not in `event_id` and cannot be parsed with this function is dropped.
```

Comment:
IME people often have a bunch of stuff in their set file events that they don't necessarily want to include, and assigning all of these somewhere can be tedious.

Also, as is, behavior is equivalent with the brainvision reader behavior.